wordle-web/src/App.vue

```diff
@@ -22,14 +22,29 @@
           <v-list>
             <v-list-item>
               <v-list-item-title>
-                <RouterLink to="/">Play Again</RouterLink>
+                <RouterLink to="/game">Play Game</RouterLink>
+              </v-list-item-title>
+            </v-list-item>
+            <v-list-item>
+              <v-list-item-title>
+                <RouterLink to="/dailywordle">Todays Word</RouterLink>
```

Comment:
```suggestion
                <RouterLink to="/dailywordle">Daily Word</RouterLink>
```
I'd try and match the route name to the navigation item name